src/Components/Facility/ConsultationDetails/index.tsx

```diff
@@ -126,6 +128,20 @@ export const ConsultationDetails = (props: any) => {
             data.symptoms_text = symptoms.join(", ");
           }
           setConsultationData(data);
+          const assetRes = await dispatch(
+            listAssetBeds({
+              bed: data?.current_bed?.bed_object?.id,
+            })
+          );
+          const isCameraAttachedRes =
+            assetRes.data.results.filter(
+              (asset: { asset_object: { meta: { asset_type: string } } }) => {
+                return asset?.asset_object?.meta?.asset_type === "CAMERA"
```

Comment:
I thought the same, but wouldn't it be simpler to have a catch all check for `asset_type` being CAMERA, instead of adding individual checks for `asset_class`? If in future we decide to add more camera brands, apart from ONVIF, the current condition will match all of them since they all would have asset_type as camera in their meta. Just a thought 🤔 

tldr: `["ONVIF","BRAND2","BRAND3","BRAND4"...].includes(asset_class)` vs `meta.asset_type === "CAMERA"`

Reply:
Do you mean `asset_class` of type `"ONVIF"` in `asset_object` ?

Reply:
yes

Reply:
I agree with @Ashesh3

Reply:
But is ONVIF a brand? Isn't it a protocol?

Reply:
I was refraining from meta, since I'm skeptical about JSON fields 😅

Reply:
> But is ONVIF a brand? Isn't it a protocol/standard?

It seems to be both 😅 On Amazon, there is a distinct brand "ONVIF" selling ONVIF cameras but I think you are right, it's an open standard first!

Reply:
> I was refraining from meta, since I'm skeptical about JSON fields 😅

I agree 😅 Although we do have strict validation for the JSON fields, it does not appear reliable to me too.